lumicks/pylake/channel.py

```diff
@@ -21,14 +22,54 @@ class Slice:
         Plot labels: "x", "y", "title".
     calibration: ForceCalibration
     """
-    def __init__(self, data_source, labels=None, calibration=None):
+    def __init__(self, data_source, labels=None, calibration=None, unit=None):
         self._src = data_source
         self.labels = labels or {}
+        self.unit = unit
         self._calibration = calibration
 
     def __len__(self):
         return len(self._src)
 
+    def _unpack_other(self, other):
+        if isinstance(other, self.__class__):
+            return other._src.data
+        else:
+            return other
+
+    def compare_unit(self, other, operation):
```

Comment:
`compare_unit` sounds like we're getting a boolean as a result, can we opt for something like the underlying `determine_unit`?

Reply:
Yep. Makes sense. Actually, it should probably be a private function, since we don't really intend for the user to mess with this directly. I changed it to _determine_unit now.